src/components/PinchZoom/_pinchZoom.scss

```diff
@@ -3,8 +3,9 @@
 .#{$px}-pinch-zoom {
   cursor: pointer;
   display: flex;
-  flex: 1;
+  margin: 0 auto !important;
```

Comment:
why can't we just use a more specific selector like the one below in line 10 to select it with a more specific selector?

Reply:
![image](https://github.com/user-attachments/assets/d379e086-50a2-42ad-a8e9-43d2fad1f439)